frontend/catalyst/jax_tracer.py

```diff
@@ -359,6 +356,33 @@ def trace_to_mlir(func, static_argnums, abstracted_axes, *args, **kwargs):
 
     # We remove implicit Jaxpr result values since we are compiling a top-level jaxpr program.
     jaxpr2, out_type2 = jaxpr_remove_implicit(jaxpr, out_type)
+
+    return jaxpr, jaxpr2, out_type2, out_tree
+
+
+def trace_to_mlir(func, static_argnums, abstracted_axes, *args, **kwargs):
+    """Lower a Python function into an MLIR module.
+
+    Args:
+        func: python function to be lowered
```

Comment:
`fn` or `func` ? let's check the consistency everywhere in the frontend

Reply:
I think this is fine.